cmd/root.go

```diff
@@ -109,7 +109,8 @@ func getRootPath() string {
 
 func setRootPath(path string) {
 	// get absolute path to .git dir (project root)
-	cmd := exec.Command("sh", "-c", "git rev-parse --show-toplevel")
+	commandArg := strings.Split("git rev-parse --show-toplevel", " ")
+	cmd := exec.Command(commandArg[0], commandArg[1:]...)
```

Comment:
We have static command here, could you please change it in this way:
```go
cmd := exec.Command("git", "rev-parse", "--show-toplevel")
```

Reply:
fixed